common-content/en/module/tools/binary-and-hexidecimal-numbers/index.md

```diff
@@ -0,0 +1,30 @@
++++
+title = "Binary and hexidecimal numbers"
+headless = true
+time = 90
+facilitation = false
+emoji= "📖"
+[objectives]
+    1="Convert numbers between any two of: binary, decimal, hexidecimal"
+    2="Explain the similarities and differences between binary, decimal, and hexidecimal numbers"
+    3="Explain why we use binary numbers"
+    4="Explain why we use hexidecimal numbers"
+    5="Describe how different programs can interpret the same bytes as different data"
+    6="Identify how many bits are needed to store integers up to some limit"
+    7="Identify the maximum integer that can be stored in a number of bits"
+    8="Add together two binary numbers without converting to decimal"
+    9="Identify which of two binary numbers is larger without converting to decimal"
+    10="Identify whether a binary number is a power of two"
++++
+
+### Binary and hexidecimal numbers
```

Comment:
```suggestion

```
You keep putting the title in but it's in the front matter so now you have it twice